daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -0,0 +1,95 @@
+- The component conformance tests have been run in a Dapr maintainers managed environment
+- The component contains a record of the conformance test result reviewed and approved by Dapr maintainers with specific components-contrib version
+- Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases
+
+### General Availability (GA)
+
+- Has at least two different users using the component in production
+- A GA component has a maintainer in the Dapr community or the Dapr maintainers
+- The component is well documented, tested and maintained across multiple versions of components-contrib repo
+
+## Conformance tests 
+
+Each component in the [components-contrib](https://github.com/dapr/components-contrib) repository needs to adhere to a set of interfaces defined by Dapr. Conformance tests are tests that are run on these component definitions with their associated backing services such that the component is tested to be conformant with the Dapr interface specifications and behavior. 
+
+The conformance tests are defined for the following building blocks: 
+
+- State store
+- Secret store
+- Bindings 
+- Pub/Sub
+
+To understand more about them see the readme [here](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md).
+
+### Test requirements
+
+- The tests should validate the functional behavior and robustness of component based on the component specification
+- All the details needed to reproduce the tests are added as part of the component conformance test documentation
+
+## Component certification process
+
+### New component certification: Alpha->Beta or Beta->GA
+
+For a new component a request for component certification follows these steps. This applies for either certifying a component going from Beta from Alpha or GA from Beta.
+- An issue is created with a request for certification of the component with the current and the new certification levels
+- A user of a component user submits a PR for integrating the component to run with the defined conformance test suite
+- The user details the environment setup in the issue created, so that a Dapr maintainer can setup the service in a managed environment
+- After the environment setup is complete, Dapr maintainers review the PR and if approved merges that PR
+- Dapr maintainers review functional correctness with the test being run in a Dapr team maintained environment
+- Dapr maintainers update the component status document categorized by Dapr Runtime version. This is done as part of the release process in the next release of Dapr runtime
+
+### Existing GA certified component
+
+For an existing GA certified component:
+- If there only internal method (not interface methods) signature changes but the behavior has not changed (for example adding new logging), then there is no need to re-certify the component
```

Comment:
This contradicts the section below where "...component has bug fixes or feature improvements". How can a change not be a fix or an improvement?